packages/block-library/src/gallery/shared.js

```diff
@@ -13,7 +13,12 @@ export function defaultColumnsNumber( imageCount ) {
 }
 
 export const pickRelevantMediaFiles = ( image, sizeSlug = 'large' ) => {
-	const imageProps = pick( image, [ 'alt', 'id', 'link' ] );
+	const imageProps = Object.fromEntries(
+		Object.entries( image ?? {} ).filter( ( [ key ] ) =>
```

Comment:
Could be `null`, so we're adding a fallback just in case:

https://github.com/WordPress/gutenberg/blob/f884657c492af84770249720a41173b0eee0cdd0/packages/block-library/src/gallery/edit.js#L178